workflow/cylc/sampled-examples/util/template.py

```diff
@@ -0,0 +1,52 @@
+# /// script
+# requires-python = ">=3.11"
+# dependencies = [
+#   "pandas",
+#   "typer",
+# ]
+# ///
+
+import pathlib
+
+import pandas
+import typer
+
+
+def main(datafile: pathlib.Path, index_column: str, row_index: str):
+    """
+    Get the cylc parameters for a preprocessing run from a CSV file.
+    """
+    df = pandas.read_csv(datafile, index_col=index_column)
+    print(_template(df, row_index))
+
+
+def _template(df, row_index):
+    """
+    Examples:
+
+        >>> import io
+        >>> csv = '''id,location,center_lat,center_lon,top_left_lat,top_left_lon,lower_right_lat,lower_right_lon,left_x,right_x,lower_y,top_y,startdate,enddate
```

Comment:
For the purposes of the example, why not include the fields needed for the template only? It'd make for a more succinct illustration.

Reply:
That's an excellent idea, done.